web/src/shared/helpers/auth-tools.ts

```diff
@@ -62,14 +63,26 @@ export const getCurrentTenant = (): string => localStorage.getItem(AUTH_CURRENT_
 
 export const refetchToken = async () => {
```

Comment:
Am I right that there was some suggestion from Ruslan to increase time we refresh this token? Should this logic be updated somewhere or not?

Reply:
> It depends on keycloak's settings.

Yep, I just didn't remember what Ruslan proposed - either unbind from keycloak's setting or just increase this default timeout. My opinion is that would be nice to keep the current solution flexible depends on the settings in keycloak (but in this case we don't need to close your PR mentioned below).